### PR TITLE
Tests reworks

### PR DESCRIPTION
**Pull request by dokshina**
* Integration tests for `version`, `start`, `stop` and `status` 
  command are rewritten in pytest
* getting-started-app tests are rewritten in pytest
* python tests are restructured: integration tests are placed
  in `test/integration`, examples tests in `test/examples`

### CODE REVIEW

File: test/utils.py [hunk 1]
```diff
@@ -0,0 +1,734 @@
+#!/usr/bin/python3
```
Comment (olegrok) [inline test/utils.py:1]: It's actually redundant

File: test/integration/test_version.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python3
```
Comment (olegrok) [inline test/integration/test_version.py:1]: It's actually redundant

File: test/integration/test_running.py [hunk 3]
```diff
@@ -0,0 +1,499 @@
+import os
```
Comment (olegrok) [inline test/integration/test_running.py:1]: There is no shebang in such file and it's OK.

File: test/examples/test_getting_started.py [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+    # get customer by wrong id
+    url = 'http://localhost:{}/storage/customers/{}'.format(router_http_port, CUSTOMER_ID+1)
+    r = requests.get(url, json=customer)
+    assert r.status_code == requests.status_codes.codes.NOT_FOUND
+    resp = r.json()
+    assert 'info' in resp
+    assert resp['info'] == 'Customer not found'
+
+    # update customer balance
+    # XXX: now I have no idea how to perform this call
```
Comment (olegrok) [inline test/examples/test_getting_started.py:168]: File an issue for each TODO

File: test/examples/test_getting_started.py [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+    # it requires account_id field, but now I don't even
+    # understand how to add an account to customer
+
+    # stop
+    cli.terminate()
```
Comment (olegrok) [inline test/examples/test_getting_started.py:173]: In case of failed test you will never terminate it. `request.addfinalizer(lambda: cli.terminate())`

Comment (dokshina) [reply test/examples/test_getting_started.py:173]: See `Cli` constructor.

Comment (olegrok) [reply test/examples/test_getting_started.py:173]: Well, then I don't understand why we should call cli.terminate twice?

Comment (dokshina) [reply test/examples/test_getting_started.py:173]: I've removed `cli.terminate` here, but generally I think that it's OK to call it twice (`terminate` defined in constructor just prevents running cli process after test fail, but here is the test emulating user session).

File: test/examples/test_getting_started.py (hunk 4 above)
Comment (dokshina) [reply test/examples/test_getting_started.py:168]: See #271

**Pull request merged**